src/app/Scheduler/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Common.Jobs;
```

Comment:
Somewhat, the configs for different envs were not merged into one level as it usually happens. Please check 🙏 

![image](https://user-images.githubusercontent.com/1188889/163795146-69bffc26-9486-4c6f-a0c2-40197c987809.png)